api/proto/teleport/trust/v1/trust_service.proto

```diff
@@ -70,3 +74,27 @@ message UpsertCertAuthorityRequest {
   // The certificate authority to create or update.
   types.CertAuthorityV2 cert_authority = 1;
 }
+
+// GenerateHostCertRequest is the request for GenerateHostCert.
+message GenerateHostCertRequest {
+  // key is the SSH public key that the certificate should include.
+  bytes key = 1;
+  // host_id is the ID of the host the certificate should include.
+  string host_id = 2;
+  // node_name is the name of the node the certificate should include.
+  string node_name = 3;
+  // principals is a list of principals that the certificate should include.
+  repeated string principals = 4;
+  // cluster_name is a list of principals that the certificate should include.
+  string cluster_name = 5;
+  // role is a system role assigned to the host that will be included on the certificate.
+  string role = 6;
```

Comment:
Looks like we would always use `types.RoleNode` for host cert signing. If that's the case, should we have this defined here? why not hardcode the role somewhere in the host cert signing ceremony?

Reply:
Since this is just a migration, I'd rather not make any behavioural changes to `GenerateHostCert` - it underpins a few key things and I wouldn't want to make an assumption about the system role without fully understanding this.